apps/dashboard/src/components/subscribers/subscriber-overview-form.tsx

```diff
@@ -237,7 +270,9 @@ export default function SubscriberOverviewForm({
               name="data"
               render={({ field, fieldState }) => (
                 <FormItem className="w-full">
-                  <FormLabel>Custom data (JSON)</FormLabel>
+                  <FormLabel tooltip="Store additional user info as key-value pairs, like address, height, or nationality, in the data field.">
+                    Custom data (JSON)
+                  </FormLabel>
```

Comment:
the form is not revalidated on change resulting in this: 

https://github.com/user-attachments/assets/4ae5c854-efe1-463c-83ff-4f6eb9d0c438